N/NEO/build_tarballs.jl

```diff
@@ -34,8 +34,14 @@ function get_script(; debug::Bool)
         ## NO
         sed -i '/-Werror/d' CMakeLists.txt
 
+        # Fails because C header is used in C++ code
+        sed -i 's/inttypes\.h/cinttypes/g' level_zero/core/source/mutable_cmdlist/mutable_indirect_data.cpp
+
         CMAKE_FLAGS=()
 
+        # Need C++20
+        CMAKE_FLAGS+=(-DCMAKE_CXX_STANDARD=20)
```

Comment:
This also sounds like a patch for upstream.  Downstream packagers should never set the C++ standard, that's the developers' business.

Reply:
That is indeed confusing 😕